scripts/generate-alerting-rtk-apis.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * To generate alerting k8s APIs, run:
+ * `npx rtk-query-codegen-openapi ./scripts/generate-alerting-rtk-apis.ts`
+ */
+
```

Comment:
maybe we should consider creating a new folder for including all the `generate-*-rtk-apis.ts`? I see that so far we are only two, but probably we will be more.
for example `scripts/rtk-gen/alerting/ .....`